### ISSUE TITLE
Canon EOS 5D Mark II profiling

### ISSUE THREAD

**Issue by mburns9** [Beep6581/RawTherapee#6318]
The Canon EOS 5D Mark II is a historically popular camera. So here is a raw shot of the classic ColorChecker 24 for profiling this camera.

https://drive.google.com/file/d/1DeMqV3a_2myUTq9Oeevi6oMyTfb8sIok/view?usp=sharing

**Comment by mburns9** [comment]
This is taken in local prenoon time at Grand Rapids, MI. The sky is mostly blue.

**Comment by Thanatomanic** [comment]
@mburns9 Sorry it took so long to get back to you about this. I have generated a DCP profile based on your image, but it is not a high quality one. This is due to two factors:
1. Foremost, the shot has raw clipping in the green channel. This likely makes interpolation in the highlights wrong.
![image](https://user-images.githubusercontent.com/6567747/130345121-3a263e4b-1f88-4724-aa05-9a2a3373831b.png)
2. It's a single illuminant profile. When your illuminant is not daylight, significant color changes can be expected. A good shot taken with a tungsten lightbulb as illuminant would be necessary to improve this.

Download here for testing: [Canon EOS 5D Mark II.zip](https://github.com/Beep6581/RawTherapee/files/7026739/Canon.EOS.5D.Mark.II.zip)
Unpack an load DCP from the Color Management tool.

Before we can consider packaging this dcp with a RawTherapee release, point 1 should be addressed. Point 2 would be very nice to have.

**Comment by mburns9** [comment]
https://drive.google.com/file/d/1af0HF7UPUlUYgWNSFGn2L4z5aWsltMXT/view?usp=sharing
Here is one taken with tungsten light from a clear bulb externally metered and exposed at EV3.

**Comment by mburns9** [comment]
The DCP profile works well. The red patch becomes red.

**Comment by mburns9** [comment]
https://drive.google.com/file/d/1Z6LY7xI_lEcYKRdeEGk8g5eFmDDqUtbU/view?usp=sharing
I found a hazy sun shot with a better histogram.

**Comment by mburns9** [comment]
https://drive.google.com/file/d/1Z6LY7xI_lEcYKRdeEGk8g5eFmDDqUtbU/view?usp=sharing
I found a hazy sun shot with a better histogram.

**Comment by mburns9** [comment]
The tungsten light raw at the link above is now improved - metered at EV2 and exposed at EV1.

**Comment by Entropy512** [comment]
Just as advice here - it sounds like you're using autoexposure?  I'd suggest manual exposure and, since most cameras don't have the ability to show a RAW histogram at capture, some bracketing - choose the most brightly exposed shot from the bracket that is not clipped.

**Comment by mburns9** [comment]
Everything is compliant to RawTherapee standards in the last two links. Nothing prevents the making of a dual-illuminant profile for this popular camera.

If I had credit here, then I could post the profile myself.

**Comment by Thanatomanic** [comment]
[Canon EOS 5D Mark II.zip](https://github.com/Beep6581/RawTherapee/files/8111919/Canon.EOS.5D.Mark.II.zip)

@mburns9 I have updated the DCP based on your new images. Please give this a testrun.

**Comment by mburns9** [comment]
https://drive.google.com/file/d/1WrhC4zpxP1353qSjWVCsGnOc68tZFM4t/view?usp=sharing

Your profile looks good to me. It makes a difference for a pink flower and a copper ceiling.

**Comment by mburns9** [comment]
https://drive.google.com/file/d/1SUWsiHvRdAtik61DgEk6TxhjW9Q5gpc5/view?usp=sharing

The copper ceiling looks better.
